src/auth-service/utils/test/ut_create-user.js

```diff
@@ -2895,4 +2895,92 @@ describe("create-user-util", function () {
       });
     });
   });
+
+  describe("syncAnalyticsAndMobile", () => {
+    it("should create a new user when the user does not exist locally", async () => {
```

Comment:
Thanks @BenjaminSsempala for this.
Awaiting for the tests for the corresponding `controller` and perhaps the routes as well but that can be ignored I guess.

Reply:
Thanks @Baalmart 
I've added those